L1Trigger/L1TMuonEndCap/src/bdt/Tree.cc

```diff
@@ -135,24 +121,14 @@ void Tree::findLeafs(Node* local_root, std::list<Node*>& tn) {
     findLeafs(local_root->getRightDaughter(), tn);
 }
 
-Tree::Tree(Tree&& tree) {
-  if (rootNode)
```

Comment:
This is the problem spotted by gcc 14. `rootNode` is uninitialized at this point so the following line deletes some random address.